packages/core/src/type-inference.ts

```
@@ -0,0 +1,505 @@
+      case "Quantified":
+        inferredType = this.inferQuantifiedType(expression);
+        break;
+      case "PositiveLookahead":
+      case "NegativeLookahead":
+        inferredType = this.inferLookaheadType(expression);
+        break;
+      case "LabeledExpression":
+        inferredType = this.inferLabeledExpressionType(expression);
+        break;
+      default:
+        const unknownType = 'type' in expression ? (expression as any).type : 'unknown';
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)